File: EasyTownHall/Backend/src/main/java/de/othr/easytownhall/models/dtos/workschedule/ServiceworkerAppointmentDTO.java
package de.othr.easytownhall.models.dtos.workschedule;

import lombok.Data;

import java.time.LocalDate;
import java.time.LocalTime;

@Data
public class ServiceworkerAppointmentDTO {
    private String departmentName;
    private String citizenName;
    private LocalTime startTime;
    private LocalDate date;
    private String reason;
}

File: EasyTownHall/Frontend/src/app/models/DTOs/Appointments/ServiceworkerAppointmentDTO.ts
export interface ServiceworkerAppointmentDTO {
    departmentName: string;
    citizenName: string;
    startTime: string; // ISO 8601 format for time, e.g., "HH:mm:ss"
    date: string; // ISO 8601 format for date, e.g., "yyyy-MM-dd"
    reason: string;
}